pkgs/development/ocaml-modules/multicore-magic/dscheck.nix

```
@@ -0,0 +1,21 @@
+{
+  lib,
+  buildDunePackage,
+  dscheck,
+  multicore-magic,
+}:
+
+buildDunePackage rec {
```

```suggestion
buildDunePackage {
```

There is no recursion here.